docs/.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["../packages/internal-tooling/eslint/index.js"],
+  "extends": ["@fluentui/internal-tooling/eslint/index.js"],
```

Comment:
Nit, should be `fluentui/internal-tooling/eslint`?

Reply:
At least for "extends" in json files it seems to need the whole path rather than doing an index import.